packages/server/jiraImagesHandler.ts

```diff
@@ -38,7 +38,8 @@ const jiraImagesHandler = uWSAsyncHandler(async (res: HttpResponse, req: HttpReq
     return
   }
 
-  const mimeType = mime.lookup(fileName)
+  const redis = getRedis()
+  const mimeType = await redis.get(`jira-image:mime-type:${fileName}`)
```

Comment:
-1 we should get the buffer and mime type at once with `redis.multi` and not having 2 roundtrips to redis. Let's use the same
```
{
      imageBuffer: Buffer,
      contentType: string
    }
```
return type in `getImageFromCache` as we use in `getImage` in `AtlassianManager`

Reply:
done